UI_UX/Frontend/src/components/Address.js

```diff
@@ -1,3 +1,31 @@
 import React from 'react';
+import { Loading } from 'carbon-components-react';
+import Jazzicon from 'react-jazzicon';
 
-export const Address = () => <div />;
+export const Address = ({ loadingAccountInfo, address }) => {
+  let loadingIcon = undefined;

```

Comment:
We don't use `let`. The value of the loading icon never changes, so just define it when you create it.

Reply:
fixed